### PR TITLE
Add china map choropleth

### PR DESCRIPTION
**Pull request by onlyvae**
## What type of PR is this? (check all applicable)

- [x] Feature

## Description
Add China map to visualization map(Choropleth)

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
图片

### CODE REVIEW

**Comment by guidopetri** [comment]
@onlyvae , thanks for the PR! We've updated a lot of things now that we're Community-driven so - if you're still interested in getting this merged - would you mind rebasing off master to re-run the CI, as well as updating merge conflicts?

We're trying to clean up our PR todo list, so if you're not interested, that's fine - we'll close the PR in about a week if we don't hear back. If you're interested in reopening the PR afterwards, we would also very much welcome that.

**Comment by justinclift** [comment]
Looking at this PR, the merge conflict is in `viz-lib/src/visualizations/choropleth/maps/countries.geo.json`.

A diff between the two shows the changes are mostly trivial (eg `United Kingdom` → `Great Britain (UK)`) with one major exception.  The new countries.geo.json added in this PR is completely missing Bahrain.

Probably the best way to fix this PR is by just removing the countries.geo.json part of it, and leaving everything else.  I'll do that now, then we can see if it passes our CI tests.

File: redash/models/users.py [hunk 1]
```diff
@@ -174,7 +174,7 @@ def profile_image_url(self):
             return self._profile_image_url
 
         email_md5 = hashlib.md5(self.email.lower().encode()).hexdigest()
-        return "https://www.gravatar.com/avatar/{}?s=40&d=identicon".format(email_md5)
+        return "https://cdn.v2ex.com/gravatar/{}?s=40&d=identicon".format(email_md5)
```
Comment (justinclift) [inline redash/models/users.py:179]: Ouch.  This is problematic as well, so lets leave this out.

**Comment by justinclift** [comment]
I've manually rebased this onto master, leaving out the `countries.geo.json` in the original PR, and also not including the commit which changed the gravatar url.

Also applied the updated formatting desired by `restyled`.

No idea at all if this will pass our CI, but if it does then we can potentially make someone review and merge it. :smile:

**Comment by justinclift** [comment]
Looks like it's failing the front-end lint tests.  Guessing it's going to need more work before it's able to be reviewed and merged then.

@onlyvae Any interest in looking at this again? :smile:

**Comment by onlyvae** [comment]
I'm very sorry, there are some unrelated commits in this PR that I mistakenly included.

**Comment by justinclift** [comment]
@onlyvae No worries.  I'm happy to adjust this PR to remove them, if you're still interested in having the map of China added? :smile:

**Pull request closed without merging**